scrapy/downloadermiddlewares/retry.py

```diff
@@ -40,19 +38,25 @@ class RetryMiddleware:
                            ConnectionLost, TCPTimedOutError, ResponseFailed,
                            IOError, TunnelError)
 
+    SETTINGS_PREFIX = ""
+    META_PREFIX = ""
+    STATS_PREFIX = ""
```

Comment:
I also wonder if, instead of allowing so customize the prefixes, we should allow to customize the actual strings.